src/syft/core/node/domain/service/request_message.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+from enum import Enum
+
+from ..... import serialize, deserialize
+from ....common import UID
+from .....proto.core.node.domain.service.request_message_pb2 import (
+    RequestMessage as RequestMessage_PB,
+)
+from ....io.address import Address
+from ....common.message import ImmediateSyftMessageWithoutReply
+from ...common.service.node_service import ImmediateNodeServiceWithoutReply
+from .....decorators import syft_decorator
+
+
+class RequestStatus(Enum):
+    Pending = 1
+    Rejected = 2
+    Accepted = 3
+
+class RequestMessage(ImmediateSyftMessageWithoutReply):
+
+    __slots__ = ["request_name", "request_description", "request_id"]
+
+    def __init__(
+        self,
+        object_id: UID,
+        address: Address,
+        owner_address,
+        request_name: str = "",
+        request_description: str = "",
+    ):
+        super().__init__(address)
+        self.request_name = request_name
```

Comment:
Let's create a Request message which subclasses from serializable and contains all the attributes relating to the request (name, description, id). That way when we change the schema we only have to change it in one place (as opposed to all the message types that handle requests)